src/openforms/forms/tests/test_data_migrations.py

```diff
@@ -0,0 +1,76 @@
+from django.db import connection
+from django.db.migrations.executor import MigrationExecutor
+from django.test import TestCase
+
+
+class TestMigrations(TestCase):
```

Comment:
please put this somewhere in `openforms.utils.tests` and update the developer documentation so that this is documented on https://open-forms.readthedocs.io/en/latest/developers/backend/core/testing-tools.html